ChronoVisor/src/VisorClientPortal.cpp

```diff
@@ -36,102 +36,93 @@ chronolog::VisorClientPortal::VisorClientPortal()
 
     // check if already started
-    if(clientPortalState != UNKNOWN)
-    { return CL_SUCCESS; }
+    if (clientPortalState != UNKNOWN) { return chronolog::CL_SUCCESS; }
 
     // TODO : keeper registry can be a member ...
     theKeeperRegistry = keeperRegistry;
-    
-    try
-    {
 
-    // initialise thalium engine for KeeperRegistryService
+    try {
+
+        // initialise thalium engine for KeeperRegistryService
 
         std::string CLIENT_PORTAL_SERVICE_NA_STRING =
-            confManager.VISOR_CONF.VISOR_CLIENT_PORTAL_SERVICE_CONF.RPC_CONF.PROTO_CONF
-            + "://" + confManager.VISOR_CONF.VISOR_CLIENT_PORTAL_SERVICE_CONF.RPC_CONF.IP
-            + ":" + std::to_string(confManager.VISOR_CONF.VISOR_CLIENT_PORTAL_SERVICE_CONF.RPC_CONF.BASE_PORT);
+                confManager.VISOR_CONF.VISOR_CLIENT_PORTAL_SERVICE_CONF.RPC_CONF.PROTO_CONF
+                + "://" + confManager.VISOR_CONF.VISOR_CLIENT_PORTAL_SERVICE_CONF.RPC_CONF.IP
+                + ":" + std::to_string(confManager.VISOR_CONF.VISOR_CLIENT_PORTAL_SERVICE_CONF.RPC_CONF.BASE_PORT);
 
         uint16_t provider_id = confManager.VISOR_CONF.VISOR_CLIENT_PORTAL_SERVICE_CONF.RPC_CONF.SERVICE_PROVIDER_ID;
 
-        margo_instance_id margo_id=margo_init(CLIENT_PORTAL_SERVICE_NA_STRING.c_str(), MARGO_SERVER_MODE, 1, 2);
- 
-        if(MARGO_INSTANCE_NULL == margo_id)
-        {
-            std::cout<<"VisorClientPortal : Failed to initialize margo_instance"<<std::endl;
-            return CL_ERR_UNKNOWN;
+        margo_instance_id margo_id = margo_init(CLIENT_PORTAL_SERVICE_NA_STRING.c_str(), MARGO_SERVER_MODE, 1, 2);
+
+        if (MARGO_INSTANCE_NULL == margo_id) {
+            std::cout << "VisorClientPortal : Failed to initialize margo_instance" << std::endl;
+            return chronolog::CL_ERR_UNKNOWN;
         }
 
-        std::cout<<"VisorClientPortal : :margo_instance initialized with NA_STRING"
-              << "{"<<CLIENT_PORTAL_SERVICE_NA_STRING<<"}" <<std::endl;
- 
-        clientPortalEngine =  new tl::engine(margo_id);
- 
-        clientPortalService = chl::ClientPortalService::CreateClientPortalService(*clientPortalEngine, provider_id, *this);
-  
+        std::cout << "VisorClientPortal : :margo_instance initialized with NA_STRING"
+                  << "{" << CLIENT_PORTAL_SERVICE_NA_STRING << "}" << std::endl;
+
+        clientPortalEngine = new tl::engine(margo_id);
+
+        clientPortalService = chl::ClientPortalService::CreateClientPortalService(*clientPortalEngine, provider_id,
+                                                                                  *this);
+
         clientPortalState = INITIALIZED;
-    
-        return_status = CL_SUCCESS;
-        
+
+        return_status = chronolog::CL_SUCCESS;
+
     }
-    catch(tl::exception const& ex)
-    {
-        std::cout<<"VisorClientPortal : Failed to start ClientPortal services"<<std::endl;
+    catch (tl::exception const &ex) {
```

Comment:
we prefer to have the brace on a new line for struct/class/function/if/for/while/try/catch. pls update your CLion code style configuration.